src/otx/algorithms/anomaly/tasks/openvino.py

```diff
@@ -216,16 +217,47 @@ def get_metadata(self) -> Dict:
         """Get Meta Data."""
         metadata = {}
         if self.task_environment.model is not None:
-            metadata = json.loads(self.task_environment.model.get_data("metadata").decode())
-            metadata["image_threshold"] = np.array(metadata["image_threshold"], dtype=np.float32).item()
-            metadata["pixel_threshold"] = np.array(metadata["pixel_threshold"], dtype=np.float32).item()
-            metadata["min"] = np.array(metadata["min"], dtype=np.float32).item()
-            metadata["max"] = np.array(metadata["max"], dtype=np.float32).item()
+            try:
+                metadata = json.loads(self.task_environment.model.get_data("metadata").decode())
+                self._populate_metadata(metadata)
+                logger.info("Metadata loaded from model v1.4.")
```

Comment:
Will this logging message require an update once otx is upgraded to 1.5+

Reply:
If we update the task then we might have to add another check that can then mention 1.5. But if we don't change the task then the model format will still be 1.4.x right?